dev_requirements.txt

```diff
@@ -0,0 +1,262 @@
+#Created by Makefile. Do not edit.
+absl-py==0.15.0
+accelerate==0.19.0
+aiobotocore==2.5.0
+aiohttp==3.8.4
+aiohttp-retry==2.8.3
+aioitertools==0.8.0
+aiosignal==1.3.1
+altair==4.2.2
+amqp==5.1.1
+antlr4-python3-runtime==4.9.3
+appdirs==1.4.4
+arff==0.9
+astunparse==1.6.3
+async-timeout==4.0.2
+asyncssh==2.13.1
+atpublic==3.1.1
+attrs==22.2.0
+backports.zoneinfo==0.2.1
+billiard==3.6.4.0
+bleach==6.0.0
+blinker==1.6.2
+blis==0.7.9
+bokeh==3.1.1
+boto3==1.26.76
+botocore==1.29.76
+cachetools==5.3.0
+catalogue==2.0.8
+celery==5.2.7
+certifi==2023.5.7
+cffi==1.15.1
+cfgv==3.3.1
+charset-normalizer==3.1.0
+click==8.1.3
+click-didyoumean==0.3.0
+click-plugins==1.1.1
+click-repl==0.2.0
+cloudpickle==2.2.1
+cmake==3.26.3
+colorama==0.4.6
+commonmark==0.9.1
+confection==0.0.4
+configobj==5.0.8
+conllu==4.5.2
+contextlib2==21.6.0
+contourpy==1.0.7
+cryptography==40.0.2
+cycler==0.11.0
+cymem==2.0.7
+Cython==0.29.34
+decorator==5.1.1
+dictdiffer==0.9.0
+dill==0.3.6
+diskcache==5.6.1
+distlib==0.3.6
+distro==1.8.0
+docker-pycreds==0.4.0
+docutils==0.15
+dpath==2.1.5
+dulwich==0.21.5
+dvc==2.56.0
+dvc-data==0.47.5
+dvc-http==2.30.2
+dvc-objects==0.22.0
+dvc-render==0.4.0
+dvc-s3==2.22.0
+dvc-studio-client==0.9.0
+dvc-task==0.2.1
+entrypoints==0.4
+et-xmlfile==1.1.0
+filelock==3.12.0
+flatbuffers==1.12
+flatten-dict==0.4.2
+flufl.lock==7.1.1
+fonttools==4.39.4
+frozenlist==1.3.3
+fsspec==2023.5.0
+funcy==2.0
+gast==0.3.3
+gensim==4.0.0
+gitdb==4.0.10
+GitPython==3.1.31
+google-auth==2.18.0
+google-auth-oauthlib==0.4.6
+google-pasta==0.2.0
+grandalf==0.8
+grpcio==1.32.0
+h5py==2.10.0
+huggingface-hub==0.14.1
+hydra-core==1.3.2
+identify==2.5.24
+idna==3.4
+importlib-metadata==4.13.0
+importlib-resources==5.12.0
+iterative-telemetry==0.0.8
+jaraco.classes==3.2.3
+jeepney==0.8.0
+Jinja2==3.1.2
+jmespath==1.0.1
+joblib==1.2.0
+jsonschema==4.17.3
+Keras-Preprocessing==1.1.2
+keyring==23.13.1
+kiwisolver==1.4.4
+kombu==5.2.4
+langcodes==3.3.0
+libpecos==0.3.0
+lit==16.0.3
+llvmlite==0.39.1
+Markdown==3.4.3
+MarkupSafe==2.1.2
+matplotlib==3.6.3
+more-itertools==9.1.0
+mpmath==1.3.0
+multidict==6.0.4
+multiprocess==0.70.14
+murmurhash==1.0.9
+nanotime==0.5.2
+nervaluate==0.1.8
+networkx==3.1
+nmslib==2.1.1
+nodeenv==1.8.0
+numba==0.56.4
+numpy==1.19.5
+nvidia-cublas-cu11==11.10.3.66
```

Comment:
these libraries do not work in my computer, i had this problem also with daniel. did you update the requirements?

Reply:
The ones you highlighted seem to have been added by Daniel in [this commit](https://github.com/wellcometrust/grants_tagger/pull/246/commits/0b6e7c12988ec2c95b41e98ab77d7057f5fe9d0f)

I didn't update the dev ones. Just ran `make update-requirements` after adding the `tensorflow-addons` in the unpnined dev requirements.

It runs fine for me. What error are you getting specifically?